src/PacketSerial.h

```diff
@@ -146,3 +147,4 @@ class PacketSerial_
 
 
 typedef PacketSerial_<COBS> PacketSerial;
+typedef PacketSerial_<SLIP, SLIP::END, 8192> SLIPPacketSerial;
```

Comment:
I'm wondering if the default buffer should be this big on an embedded system.  Perhaps leave it default -- something like:

```c++
 typedef PacketSerial_<COBS> PacketSerial;
 typedef PacketSerial_<COBS> COBSPacketSerial;
 typedef PacketSerial_<SLIP, SLIP::END> SLIPPacketSerial;
```